go.mod

```diff
@@ -77,6 +77,7 @@ require (
 	github.com/containerd/typeurl v1.0.0
 	github.com/coreos/go-semver v0.3.0
 	github.com/coreos/go-systemd v0.0.0-20190620071333-e64a0ec8b42a
+	github.com/davecgh/go-spew v1.1.1
```

Comment:
It doesn't look like we are using it anymore in this PR, do we need it?

Reply:
It is used in `pkg/security/secl/generators/accessors/accessors.go`